ingestion/tests/unit/topology/database/test_datalake.py

```diff
@@ -571,8 +571,8 @@ def test_avro_file_parse(self):
                         }
                     }
                 },
-                "bucketName": "bucket name",
-                "prefix": "prefix",
+                # "bucketName": "bucket name",
```

Comment:
why is this commented?

Reply:
Thanks! Had commented it out to test one thing and forgot to fix it. It should be changed, not commented!